### PR TITLE
[HER-129] Enhance camera filters

### PR DESCRIPTION
**Pull request by ConstantinescuAndrei**
## Description
- Add latitude and longitude filter
- Create latitude and longitude models and types
- Change camera filter

## Related
[HER-129](https://nerds-interns.atlassian.net/jira/software/projects/HER/boards/1?selectedIssue=HER-129)

### CODE REVIEW

File: src/Presentation/Camera/Fetching/Types/TPCamera.cs [hunk 1]
```diff
@@ -11,8 +11,13 @@ public class TPCamera : InputObjectGraphType<PCamera>
     {
         public TPCamera()
         {
-            Field(camera => camera.Ids, true).Description("This is the list of ids");
-            Field(camera => camera.Pagination, false, typeof(TPPagination)).Description("This is the pagination.");
+            Name = "CameraParameter";
+            Description = "Cameras will be filtered by this fields";
+
+            Field(camera => camera.Ids, true).Description("If no value will be given, then the cameras will not be filtered by id.");
```
Comment (danidoff) [inline src/Presentation/Camera/Fetching/Types/TPCamera.cs:17]: Should we move the description to the next line and also a new line between the fields? (for all the create types)

File: src/Presentation/Camera/Fetching/Types/TPLatitude.cs [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    [SuppressMessage(Category.Default, Check.CA1724, Justification = Reason.Readability)]
+    public class TPLatitude : InputObjectGraphType<PLatitude>
+    {
+        public TPLatitude()
+        {
+            Name = "LatitudeFilter";
+            Description = "This will be used for filtering the cameras by latitude";
+
+            Field(latitude => latitude.LesserEqualThan, true).Description("Cameras will have the latitude lesser or equal than this value.");
+            Field(latitude => latitude.GreaterEqualThan, true).Description("Cameras will have the latitude greater or equal than this value.");
```
Comment (danidoff) [inline src/Presentation/Camera/Fetching/Types/TPLatitude.cs:17]: Also here, please move the description to the next line and space between the fields.

File: src/Presentation/Camera/Fetching/Types/TPLongitude.cs [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+    [SuppressMessage(Category.Default, Check.CA1724, Justification = Reason.Readability)]
+    public class TPLongitude : InputObjectGraphType<PLongitude>
+    {
+        public TPLongitude()
+        {
+            Name = "LongitudeFilter";
+            Description = "This will be used for filtering the cameras by longitude";
+
+            Field(longitude => longitude.LesserEqualThan, true).Description("Cameras will have the longitude lesser or equal than this value.");
```
Comment (danidoff) [inline src/Presentation/Camera/Fetching/Types/TPLongitude.cs:16]: Also here, please move the description to the next line and space between the fields.

**Review by danidoff — CHANGES_REQUESTED** [review]

File: src/Presentation/Camera/Fetching/Types/TPLongitude.cs (hunk 3 above)
Comment (ConstantinescuAndrei) [reply src/Presentation/Camera/Fetching/Types/TPLongitude.cs]: done

File: src/Presentation/Camera/Fetching/Types/TPLatitude.cs (hunk 2 above)
Comment (ConstantinescuAndrei) [reply src/Presentation/Camera/Fetching/Types/TPLatitude.cs]: done

File: src/Presentation/Camera/Fetching/Types/TPCamera.cs (hunk 1 above)
Comment (ConstantinescuAndrei) [reply src/Presentation/Camera/Fetching/Types/TPCamera.cs]: done

**Review by danidoff — APPROVED** [review]

**Review by TBiannca — APPROVED** [review]

**Review by Urzica-Platon-Casian — APPROVED** [review]
Sexy!

**Review by RaduBuciuceanu — APPROVED** [review]

**Pull request merged**